Add bot account article (#3725)
ppy/osu-wiki: wiki/Help_Centre/Bot_Account/en.md

> ---
> needs_cleanup: false
> ---
> 
> # Bot Account criteria

Reviewer: ```suggestion
# Bot Account criteria
```
if the article does not require any clean-up, there is no need to specifically mention it here, afaik

> Some users have been creating IRC bots with various functionalities that they run on their own accounts. We offer the possibility to create an account specifically for the bot.

Reviewer 2: ```suggestion
Some users have been creating "chat bots" with various functionalities offered to the community via multiplayer lobbies or PM converstions. This is generally permitted as long as it is in good taste.

If you are looking to start a similar service, please begin by running it on your own osu! account. As it grows, there may be an opportunity to request an account specifically for the bot to live on.
```

> **Do not create the bot account before checking with us first. This will be considered multi-accounting and be punished.**

Reviewer 2: ```suggestion
**Do not create a bot account before checking with us first. This will be considered multi-accounting and be punished.**
```

- The bot must be fully open source and have full public documentation
- The bot must be used by at least 50 unique users every month
- The bot must respect the rate limits (on regular accounts at first)
- **The bot must not send any messages in non-multiplayer public channels.** All bot interaction must be done in private messages.
- The bot must be of use by the community. Personal bots will not be granted their own accounts.

## Applying for a bot account

If your bot fulfills the criteria above, you can file a request to create an account specifically for it.

Send an email to [accounts@ppy.sh](mailto:accounts@ppy.sh) with "Bot Account Request: nameofyourbot" in the title. This must be sent from the email address attached to your account.

The body of the email should contain the following:
- Date on which you began running the bot
- A link to the source code and documentation
- A short summary of what your bot does and its functionalities

We will review the request and decide whether or not to grant you permission to create an account for the bot. If your request is denied, we will let you know why.

## Bot Account Creation

> If your request is approved, you will receive a reply allowing you to create a new account for the bot. **Do not submit any plays on the account** and email us back with the account's username and link. We will tell you when you can migrate your bot to the account.

Reviewer 2: we will make the account for them. they should provide the username only.

> **The bot account must never have any plays submitted on it.** If we catch a bot account being played on, it will be immediately terminated.

Reviewer 2: unnecessary, we can do this automatically.
